### PR TITLE
Better explanation of jwt. Reorganizing files, adding links.

### PR DESCRIPTION
**Pull request by matthiashanel**
Signed-off-by: Matthias Hanel <mh@synadia.com>

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -79,14 +79,17 @@
       * [Username/Password](nats-server/configuration/securing_nats/auth_intro/username_password.md)
       * [TLS Authentication](nats-server/configuration/securing_nats/auth_intro/tls_mutual_auth.md)
       * [NKeys](nats-server/configuration/securing_nats/auth_intro/nkey_auth.md)
-      * [Accounts](nats-server/configuration/securing_nats/auth_intro/accounts.md)
-      * [JWTs](nats-server/configuration/securing_nats/auth_intro/jwt_auth.md)
       * [Authentication Timeout](nats-server/configuration/securing_nats/auth_intro/auth_timeout.md)
     * [Authorization](nats-server/configuration/securing_nats/authorization.md)
+    * [Multi Tenancy using Accounts](nats-server/configuration/securing_nats/accounts.md)
```
Comment (gcolliso) [inline SUMMARY.md]: The Multi-Tenancy link is broken nats-server/configuration/securing_nats/accounts.md

Comment (matthiashanel) [reply SUMMARY.md]: Hey Ginger, are you sure? I moved the entire file.

**Comment by aricart** [comment]
Is there a way where we can preview the site? it is very painful just looking at the markdown files. The old `make` script doesn't seem to work, perhaps we can update it so that we can generate a preview without having to know the detailed steps to build and preview the site.

**Comment by matthiashanel** [comment]
@aricart , Hey Alberto. I'm not aware of a way.
What we could attempt at some time is to use a formatter or fix all linter warnings.
Part of what makes this painful are the long lines...

**Comment by gcolliso** [comment]
> Is there a way where we can preview the site? it is very painful just looking at the markdown files. The old `make` script doesn't seem to work, perhaps we can update it so that we can generate a preview without having to know the detailed steps to build and preview the site.

Unfortunately, GitBook doesn't allow a preview. I just use the preview on Github to view the changes. I've put in an ER for this but there is no ETD.

File: nats-tools/nas/README.md [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
+The server supports an number of stores which enable it to serve account [JWTs](../../nats-server/configuration/securing_nats/jwt/README.md) from:
 
-* a directory
-* an [NSC](../nsc/nsc.md) directory
-* memory \(for testing purposes\)
+* a [directory](nas_conf.md#directory-configuration)
+* an [NSC](../nsc/nsc.md) [directory](nas_conf.md#nsc-configuration)
 
-The server can operate in a _READ ONLY_ mode where it serves content from a directory, or in notification mode, where it can notify a NATS server that a JWT in the store has been modified, updating the NATS server with the updated JWT.
+> For testing purposes the nats server can be configured with a [memory resolver](../../nats-server/configuration/securing_nats/jwt/resolver.md#memory) that avoids usage of the account server.
```
Comment (wallyqs) [inline nats-tools/nas/README.md:9]: memory resolver can also be a valid option for prod, I'd leave out the `testing purposes` part

File: nats-tools/nas/dir_store.md [hunk 3]
```diff
@@ -100,3 +100,4 @@ Quick check on nsc to verify the ids of the accounts on nsc, match the files:
 
 Note that the `@` before the file name is required for `curl` to read the specified file, and use it as the payload. Otherwise, it will simply post the path specified, which will result in an update error.
 
+Curl can also be used to [inspect](inspecting_jwts.md) JWTs.
```
Comment (wallyqs) [inline nats-tools/nas/dir_store.md:103]: Maybe can add a curl example, there is a decode pretty option as well I think

Comment (wallyqs) [reply nats-tools/nas/dir_store.md:103]: ah ok  thanks

**Pull request merged**